### DISCUSSION TITLE
Galaxy Brain Achievement

### DISCUSSION THREAD

**Discussion by Mimi-Dang** [peterdanwan/gimme_readme#17] [Q&A] [answered]
How do I get the Galaxy Brain Achievement... please Peter

**Comment by peterdanwan** [comment, marked as answer]
Hi Mimi, to get a `Galaxy Brain Achievement`, you need to respond to a `Q&A Discussions post` and hope that the user who made the `Q&A Discussions post`  marks your post with the `Marked as Answer` flag.
